scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/activities/ActivityExpression.java

```diff
@@ -609,7 +381,6 @@ public String prettyPrint(final String prefix) {
   @Override
   public void extractResources(final Set<String> names) { }
 
-
   /**
    * Evaluates whether a SerializedValue can be qualified as the subset of another SerializedValue or not
     * @param superset the proposed superset
```

Comment:
Minor, can ignore: A little below here on L402, 408, 417: 
The returns can be cleaned up slightly (using Numeric as an example):
```java
return argumentsAsNumeric.map(bigDecimal -> bigDecimal.equals(value))
                         .orElse(false);
```